pod/main/static/css/pod.css

```diff
@@ -664,6 +664,133 @@ div.card a img {
   width: 100%;
 }
 
+/** Dashboard **/
+#bulk_update_container{
+    background-color: var(--pod-background-neutre2-bloc);
+    padding: 1em;
+    border-radius: var(--bs-border-radius);
+}
+.btn-dashboard-display-mode{
+    height: fit-content;
+}
+#countSelectedVideosBadge{
+  background-color: var(--pod-background-neutre1-bloc);
+  color: var(--pod-font-color);
+}
+/* Dashboard infinite container */
+.infinite-container{
+  margin: 1rem 0;
+}
+.infinite-container .infinite-item{
+    cursor: pointer;
+}
+.pod-infinite-container .infinite-item{
+    position: relative;
+}
+.infinite-item .checked_overlay{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: none;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 0;
+    z-index: 9;
```

Comment:
Je propose de simplifier `left,right,bottom,top` par :
`inset: 0;`
cf https://caniuse.com/mdn-css_properties_inset

Reply:
not yet